### ISSUE TITLE
Edit Experience & Skills page with Required Skills

### ISSUE THREAD

**Issue by tristan-orourke** [GCTC-NTGC/gc-digital-talent#3326]
# Description

[Figma Prototype](https://www.figma.com/proto/N9NoPunkqaynjsi89Wrbh0/Direct-Intake---Poster-and-application?node-id=138%3A2829&starting-point-node-id=226%3A4567&scaling=min-zoom)

Create a version of the Experience and Skills page which also tells you which skills you are missing (considering the pool you are applying to). Transferable skills are not considered required and should not be listed.

Note: The Transferable/Occupational category comes from the Skill Family, which skills may have several of. If a skill has zero skill families, list it here. Only  Consider a skill to be Transferable (and therefore unlisted) if it has at least one SkillFamily and all are Transferable.

![image](https://user-images.githubusercontent.com/4313717/178765483-ef89e176-953a-43e2-8a8d-b53315f1c409.png)

# Acceptance Criteria

- [x] this version of the Experience and Skills page is rendered if the url has an applicationId query parameter
- [ ] when creating or editing an Experience from this page, the url and breadcrumbs are relative this one, and saving returns you here
- [x] All required and Nice to Have non-transferrable skills are listed. 
- [x] Transferable skills associated with the pool are not listed.

**Comment by petertgiles** [comment]
No null state - section disappears if all skills are satisfied

**Comment by petertgiles** [comment]
Section is non-interactive

**Comment by petertgiles** [comment]
Calculation of missing skills should happen in the frontend

**Comment by brindasasi** [comment]
blocked by Hydrogen PR
